From 5d886672afddd4ac7103da0fdd09c538544a7e22 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Wed, 6 May 2020 12:57:30 +0200
Subject: [PATCH 697/830] ANDROID: Drop ABI monitoring from KASAN build config
X-NVConfidentiality: public

That should not be enabled nor tracked for the KASAN config.

Fixes: 5d52e246c9c3 ("ANDROID: Rename build.config.gki.arch_kasan")
Signed-off-by: Matthias Maennich <maennich@google.com>
Change-Id: I0d2fa1748bcfd844667f4b8c9e013d2fd24bca05
---
 build.config.gki_kasan.aarch64 | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/build.config.gki_kasan.aarch64 b/build.config.gki_kasan.aarch64
index 9099658338a6..3ce42b72ffd2 100644
--- a/build.config.gki_kasan.aarch64
+++ b/build.config.gki_kasan.aarch64
@@ -1,11 +1,3 @@
 . ${ROOT_DIR}/common/build.config.common
 . ${ROOT_DIR}/common/build.config.aarch64
 . ${ROOT_DIR}/common/build.config.gki_kasan
-
-ABI_DEFINITION=abi_gki_aarch64.xml
-KMI_WHITELIST=abi_gki_aarch64_whitelist
-ADDITIONAL_KMI_WHITELISTS="
-abi_gki_aarch64_cuttlefish_whitelist
-abi_gki_aarch64_qcom_whitelist
-"
-TRIM_NONLISTED_KMI=1
-- 
2.17.1

